research.html

```diff
@@ -0,0 +1,194 @@
+        .video-container video {
+            width: 100%;
+        }
+    }
+    </style>
+</head>
+
+<body>
+    <div class="layout">
+        <main id="content" class="content">
+            <section class="heavy">
+                <div class="container">
+                    <div class="col-3">
+                        <div class="hero">
+                            <a href="/"><img src="/logos/logov3.svg" alt="Unitary Fund" /></a>
+                            <p class="tagline"><b>Because evolution is <a
+                                        href="https://en.wikipedia.org/wiki/Unitarity_(physics)"
+                                        target="_blank">unitary</a></b>.</p>
+                            <button id="mobile-menu-icon" class="hamburger hamburger--squeeze" type="button">
+                                <span class="hamburger-box">
+                                    <span class="hamburger-inner"></span>
+                                </span>
+                            </button>
+                        </div>
+                        <p class="subtitle leading-arrow"><a name="research">Research with Unitary Fund</a></p> Unitary Fund believes that research is important for quantum open source. We focus not only on our own research interests, but also building tools to enable others, like those in our microgrant program. Check out some of the many research projects we've been involved in or supported below!
+                        <p class="subtitle leading-arrow"><a name="research-uf">Unitary Labs research and collaborations</a></p>
+                        The Unitary Fund research <a href="https://github.com/unitaryfund/research">GitHub repo</a> has source code for a variety of our research projects.
+                        <p>
+                            <ul style="list-style-type: none;">
+                                <li class="leading-block"><a href="https://arxiv.org/abs/2009.04417">Mitiq: A software package for error mitigation on noisy quantum computers</a>, <a href="https://mitiq.readthedocs.io/en/latest/examples/mitiq-paper/mitiq-paper-codeblocks.html">source code</a></li><br>
+                                <li class="leading-block"><a href="https://arxiv.org/abs/2005.10921">Digital zero noise extrapolation for quantum error mitigation</a></li><br>
+                                <li class="leading-block"><a href="https://arxiv.org/abs/2108.02237">Extending quantum probabilistic error cancellation by noise scaling</a></li><br>
+                                <li class="leading-block"><a href="https://arxiv.org/abs/2105.09902">Pulse-level noisy quantum circuits with QuTiP</a></li><br>
+                                <li class="leading-block"><a href="https://arxiv.org/abs/2104.15044">Pulser: An open-source package for the design of pulse sequences in programmable neutral-atom arrays
+                                </a></li><br>
+                            </ul>
+                        </p>
```

Comment:
```suggestion
                        <p class="subtitle leading-arrow"><a name="research-uf">Unitary Labs research and collaborations</a></p>
                        <p>
                            <ul style="list-style-type: none;">
                                <li class="leading-block"><a href="https://arxiv.org/abs/2009.04417">Mitiq: A software package for error mitigation on noisy quantum computers</a>, <a href="https://mitiq.readthedocs.io/en/latest/examples/mitiq-paper/mitiq-paper-codeblocks.html">source code</a></li><br>
                                <li class="leading-block"><a href="https://arxiv.org/abs/2005.10921">Digital zero noise extrapolation for quantum error mitigation</a></li><br>
                                <li class="leading-block"><a href="https://arxiv.org/abs/2108.02237">Extending quantum probabilistic error cancellation by noise scaling</a></li><br>
                                <li class="leading-block"><a href="https://arxiv.org/abs/2105.09902">Pulse-level noisy quantum circuits with QuTiP</a></li><br>
                                <li class="leading-block"><a href="https://arxiv.org/abs/2104.15044">Pulser: An open-source package for the design of pulse sequences in programmable neutral-atom arrays
                                </a></li><br>
                            </ul>
                        The Unitary Fund research <a href="https://github.com/unitaryfund/research">GitHub repo</a> has source code for a variety of our research projects.
                        </p>
```

Please ignore this suggestion if don't like it. When reading this part it sounded like we had source code for all the papers listed after the first sentence. So the suggestion would be to to mention the research GitHub repo after the list of papers.

Reply:
Yeah good point. I guess my logic was "variety of projects" didn't mean all of them and having it first increased the likelihood they would click on it. Let me try tweaking the wording on this so having it first sets expectations accordingly.